config/swagger.yaml

```diff
@@ -3162,7 +3684,49 @@ definitions:
         description: 'Blocks''s height'
     required:
     - height
-
+  Status:
+    type: object
+    properties:
+      genesis-key-block-hash:
+        $ref: '#/definitions/EncodedHash'
+      solutions:
+        type: integer
+        format: int64
+        minimum: 0
+      difficulty:
+        type: integer
+        format: int64
+      syncing:
+        type: boolean
+      listening:
+        type: boolean
+      protocols:
+        type: array
+        items:
+          $ref: '#/definitions/Protocol'
+      node-version:
+        type: string
+      node-revision:
+        type: string
+      peer-count:
+        type: integer
+        format: int64
+        minimum: 0
+      pending-transactions-count:
+        type: integer
+        format: int64
+        minimum: 0
+  Protocol:
+    type: object
+    properties:
+      protocol_version:
+        type: integer
+        format: int64
```

Comment:
From time to time check Python UAT tests run - so not to use too advanced features that tools do not support. (No obvious issue with this line - this is just a reminder.)

Reply:
✅

Reply:
BTW in [recent master](https://github.com/aeternity/epoch/commit/f735a5b0016531206f7ce5c8c63002c563629ec9) Python UAT tests run on each branch.